core/src/macros.rs

```diff
@@ -11,19 +10,19 @@ macro_rules! cfg_feature {
 
 macro_rules! cfg_client {
 	($($item:item)*) => {
-		$crate::cfg_feature!("client", $($item)*);
+		cfg_feature!("client", $($item)*);
```

Comment:
I wasn't sure, but things seem to work fine without properly namespacing the macro!

Reply:
Hmm, here be dragons. I *think* there are situations when this will not work well, but can't remember what the full story was. `$crate::macros::cfg_feature!` should work though right?